app.js

```diff
@@ -1,43 +1,50 @@
-const createError = require('http-errors');
```

Comment:
why we are not using single quotes , I think we should use single quotes. You can enable eslint already I have checked In .eslintrc file you can enable on your dev tool

Reply:
if you are referring to the quotes around http-error then they are already single quotes

Reply:
I am referring to all quotes wherever we are using in this application. nopes `http-error` is in single quotes.

Reply:
this problem is due to react plugins that i am using which have norms according to react on autosave they changes will see to it